alternator/ttl.cc

```diff
@@ -354,96 +537,13 @@ future<> expiration_service::run() {
             db::consistency_level cl = db::consistency_level::LOCAL_QUORUM;
             auto query_options = std::make_unique<cql3::query_options>(cl, std::vector<cql3::raw_value>{});
             query_options = std::make_unique<cql3::query_options>(std::move(query_options), std::move(paging_state));
-            dht::partition_range_vector partition_ranges;
-            partition_ranges.push_back(dht::partition_range::make_open_ended_both_sides());
-            auto p = service::pager::query_pagers::pager(s, selection, *query_state_ptr, *query_options, command, std::move(partition_ranges), nullptr);
-            while (!p->is_exhausted()) {
```

Comment:
Please fold into the patch that introduced this.

Reply:
If I understand you correctly I assume you mean fold the patch which introduced scan-everything-on-each-shard with this patch that scans different ranges on each shard. Ok, will do.

Reply:
Combined the two patches.